### DISCUSSION TITLE
Add Instant Sell Moving Coins and Instant Buy Moving Coins to the API

### DISCUSSION THREAD

**Discussion by csjh** [HypixelDev/PublicAPI#455] [Ideas]
I noticed in the in-game bazaar graph portion there's an option to look at the total coins gained from instantly selling and total coins spent on instantly buying (respectively). Since it's already implemented in game, would it be possible to add these stats to the API? If not solely because if we're being honest, the in-game graphs aren't exactly a pleasure to look at. 

Thanks!

**Comment by csjh** [comment]
Any chance of this happening? Would be incredibly useful.

**Comment by Semisol** [comment]
Bumping this.
